ios/App/App/plugins/AbsAudioPlayer.swift

```diff
@@ -35,15 +34,14 @@ public class AbsAudioPlayer: CAPPlugin {
     }
     
     func restorePlaybackSession() async {
-        // We don't need to restore if we have an active session
-        guard PlayerHandler.getPlaybackSession() == nil else { return }
-        
```

Comment:
We actually do need to restore, in the case of the WebKit reloading and existing playback session is already in the player.